wrappers/swift/Sources/SwiftWrapper/EventProperties.swift

```diff
@@ -3,43 +3,23 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import ObjCModule
+
 /**
 Represents Event's properties.
 */
-public class EventProperties {
-    /// Event Name.
-    private var name: String
-    /// Event priority.
-    private var priority: ODWEventPriority = ODWEventPriority.unspecified
-    /// Event properties, Key being the name of the property and Value is property value.
-    private var _properties: [String: Any] = [:]
-    var properties: [String: Any] {
-        get {
-            return Dictionary(uniqueKeysWithValues: _properties.lazy.map { ($0, $1) })
-        }
-    }
-    /// Event PII (Personal Identifiable Information) tags, Key being property name and Value is PIIKind value.
-    private var _piiTags: [String: ODWPiiKind] = [:]
-    var piiTags : [String: ODWPiiKind] {
-        get {
-            return Dictionary(uniqueKeysWithValues: _piiTags.lazy.map { ($0, $1) })
-        }
-    }
-    /// Base type of an event.
-    private var eventType: String = String()
-
-    private func setPiiTag(_ name: String, withPiiKind piiKind: ODWPiiKind) {
-         self._piiTags[name] = piiKind
-    }
+public final class EventProperties {
+    /// Obj-C class object which swift is wrapped around.
+    private var odwEventProperties: ODWEventProperties
 
     /**
     Constructs EventProperties object with event name.
 
     - Parameters:
-        - name: Name of the event.
+        - name: Name of the event as a `String`.
     */
-    public init(name: String!) {
-        self.name = name
+    public init(name: String) {
+        self.odwEventProperties = ODWEventProperties(name: name)
```

Comment:
In Swift, using `self.` outside of a closure is considered anti-pattern. We should probably remove these explicit `self.`s to respect Swift best practices and conventions.